test/test_nn.py

```diff
@@ -4635,6 +4635,32 @@ def test_data_parallel_device_args(self):
         out = dp.data_parallel(l, i, device_ids=(cuda0, cuda1), output_device=cuda0)
         self.assertEqual(out, l(i))
 
+    @unittest.skipIf(not TEST_MULTIGPU, "multi-GPU not supported")
+    @skipIfRocm
+    def test_data_parallel_function_deletion(self):
```

Comment:
Why is this related to this PR?

Reply:
This tests the original bug that spawned this line of inquiry :)